server/datastore/mysql/migrations/tables/20240521143023_CreateTableMDMAssets.go

```diff
@@ -0,0 +1,38 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240521143023, Down_20240521143023)
+}
+
+func Up_20240521143023(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE mdm_config_assets (
+    id int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY, 
+
+    -- name is used for humans to identify what value is stored in this row
+    name varchar(256) COLLATE utf8mb4_unicode_ci NOT NULL DEFAULT '',
+
+    -- value holds the raw value of the asset
+    value blob NOT NULL,
```

Comment:
Just verifying space-wise if we're ok with `blob`, which follows the same (unintuitive, if you ask me) size limits as for `text`, so 2^16 bytes for `blob` vs bigger `mediumblob` and `largeblob`.

Reply:
thanks, I will increase it, I think it's worth it given that this table will have very few rows anyways.